subsys/esb/esb.c

```diff
@@ -1672,6 +1694,10 @@ int esb_init(const struct esb_config *config)
 
 	nrf_radio_fast_ramp_up_enable_set(NRF_RADIO, esb_cfg.use_fast_ramp_up);
 
+#if defined(CONFIG_ESB_FAST_CHANNEL_SWITCHING)
```

Comment:
IS_ENABLED ?

Reply:
RXREADY event not available to all SoCs.